packages/horizon-add-document/handler.js

```diff
@@ -63,12 +58,25 @@ async function parseFile(log, documentId, applicationId, documentType) {
   };
 }
 
+async function parseFile({ log, body }) {
+  const { documentId, applicationId, documentType } = body;
+  log.info({ applicationId, documentId, documentType }, 'Downloading document');
+
+  const { data } = await axios.get(`/api/v1/${applicationId}/${documentId}/file`, {
+    baseURL: config.documents.url,
+    params: {
+      base64: true,
+    },
+  });
+
+  return createDataObject(data, body);
+}
+
 module.exports = async (event, context) => {
   event.log.info({ config }, 'Receiving add document request');
 
   try {
-    const { body } = event;
-    const { caseReference, applicationId, documentId, documentType } = body;
```

Comment:
weren't these params needed?

Reply:
I've done some boy scout jiggery pokery to improve readability, passing data objects instead of multiple params, basically.